changelog/10394.improvement.md

```diff
@@ -0,0 +1,12 @@
+Allow multiple entities to be annotated for the same word/tokens.  
+When using entity extractors that support generating multiple entities from a single expression, the test stories fail as there is no way to annotate multiple entity_types and entity_values.
+Entity Extractors like DIET are not optimized for training with multiple entity extractions, so be sure to use only Regex or FlashText or similar extractors.  
```

Comment:
```suggestion
Allow single tokens in rasa end-to-end test files to be annotated with multiple entities.

Some entity extractors (s.a. `RegexEntityExtractor`) can generate multiple entities from a single expression. Before this change, `rasa test` would fail in this case, because test stories could not be annotated correctly.
```